scipy/optimize/_highs/src/HConfig.h.in

```diff
@@ -0,0 +1,20 @@
+#ifndef HCONFIG_H_
```

Comment:
This file isn't used, remove? Or is there an attempt here to keep things buildable with CMake? If so, would be good to explain in a comment at the top of `setup.py`.

Reply:
Removed